includes/ApplHandlingByStr.php

```diff
@@ -82,6 +86,22 @@ function showFullStatus( $stat_str )
 }
 
 
+function findPending( $stat_str )
+{
+	$STR = explode ( ';', $stat_str );
+	for ( $i =  1;  $i < count ( $STR ) ; $i++ )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space found after opening bracket of FOR loop
* Space found before closing bracket of FOR loop
* Expected 1 space after first semicolon of FOR loop; 2 found
* Space found before second semicolon of FOR loop
* Expected 0 spaces after opening bracket; 1 found
* Space before opening parenthesis of function call prohibited
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 5

<!-- Comment by [SideCI](https://sideci.com) -->